test/models/work_test.rb

```diff
@@ -0,0 +1,61 @@
+require "test_helper"
+
+describe Work do
+  describe "validations" do
+    before do
+      @work = Work.new(title: "test work", category: "album")
+    end
+
+    it "is valid when title and category are present" do
+      result = @work.valid?
+      expect(result).must_equal true
+    end
+
+    it "is invalid if title is nil" do
+      @work.title = nil
+      result = @work.valid?
+      expect(work.category).must_equal "album"
```

Comment:
This test produced an error. Looks like you're missing an `@`. Whoops!
```suggestion
      expect(@work.category).must_equal "album"
```